app/bundles/UserBundle/Model/UserModel.php

```diff
@@ -391,4 +391,53 @@ public function getOwnerListChoices(): array
     {
         return $this->getRepository()->getOwnerListChoices();
     }
+
+    public function createInvite(string $email, int $roleId): UserInvite
+    {
+        $role = $this->em->getRepository(Role::class)->find($roleId);
+        if (!$role) {
+            throw new \InvalidArgumentException($this->translator->trans('mautic.user.invite.error.invalid_role', [], 'validators'));
+        }
+
+        $token  = bin2hex(random_bytes(32));
+        $invite = (new UserInvite($role))
+            ->setEmail($email)
+            ->setToken($token)
+            ->setExpiration((new \DateTime())->add(new \DateInterval('PT48H')));
+        $this->em->persist($invite);
+        $this->em->flush();
+
+        $link   = $this->router->generate('mautic_user_invite_register', ['token' => $token], UrlGeneratorInterface::ABSOLUTE_URL);
+        $mailer = $this->mailHelper->getMailer();
+        $mailer->setTo([$email => $email]);
+        $mailer->setSubject($this->translator->trans('mautic.user.invite.subject'));
+        $text = $this->translator->trans('mautic.user.invite.email.body', ['%invite_link%' => '<a href="'.$link.'">'.$link.'</a>']);
+        $text = str_replace('\\n', "\n", $text);
+        $html = nl2br($text);
+        $mailer->setBody($html);
```

Comment:
Should we create a default email template (Twig) with some standard branding, including header, content, and footer?